app/Http/Requests/PostReplyRequest.php

```diff
@@ -24,7 +24,7 @@ public function authorize()
     public function rules()
     {
         return [
-            'content' => 'required',
+            __('content') => 'required',
```

Comment:
@milack68 Does the rules fields need to be translated? Is the validation still running when using the translation?

Reply:
Yes - the validation is still running and the rule-fields are translated.